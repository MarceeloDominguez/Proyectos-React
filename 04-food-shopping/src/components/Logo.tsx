import Image from "next/image";
import Link from "next/link";

type Prop = {
  width: number;
  height: number;
};

export default function Logo({ width, height }: Prop) {
  return (
    <Link href={"/"}>
      <Image src="/logo.svg" alt="Logo" width={width} height={height} />
    </Link>
  );
}
