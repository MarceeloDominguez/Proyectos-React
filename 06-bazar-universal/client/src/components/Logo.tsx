import Link from "next/link";

export default function Logo() {
  return (
    <Link href="/">
      <p className="font-bold text-[#53B175] text-lg">T-Online</p>
    </Link>
  );
}
