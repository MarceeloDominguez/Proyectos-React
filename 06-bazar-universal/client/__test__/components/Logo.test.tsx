import { render } from "@testing-library/react";
import Logo from "@/components/Logo";

describe("<Logo />", () => {
  it("Should render the logo", () => {
    const { container } = render(<Logo />);
    expect(container).toMatchSnapshot();
  });
});
