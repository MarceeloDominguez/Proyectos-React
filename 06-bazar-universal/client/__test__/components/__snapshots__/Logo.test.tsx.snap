// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <a
    href="/"
  >
    <p
      class="font-bold text-[#53B175] text-lg"
    >
      T-Online
    </p>
  </a>
</div>
`;

exports[`<Logo /> Should render the logo 1`] = `
<div>
  <a
    href="/"
  >
    <p
      class="font-bold text-[#53B175] text-lg"
    >
      T-Online
    </p>
  </a>
</div>
`;

exports[`<Logo /> render logo 1`] = `
<div>
  <a
    href="/"
  >
    <p
      class="font-bold text-[#53B175] text-lg"
    >
      T-Online
    </p>
  </a>
</div>
`;

exports[`<Logo /> should render the logo 1`] = `
<div>
  <a
    href="/"
  >
    <p
      class="font-bold text-[#53B175] text-lg"
    >
      T-Online
    </p>
  </a>
</div>
`;

exports[`Logo  1`] = `
<div>
  <a
    href="/"
  >
    <p
      class="font-bold text-[#53B175] text-lg"
    >
      T-Online
    </p>
  </a>
</div>
`;

exports[`Logo render logo 1`] = `
<div>
  <a
    href="/"
  >
    <p
      class="font-bold text-[#53B175] text-lg"
    >
      T-Online
    </p>
  </a>
</div>
`;

exports[`Logo render logo and show the text 1`] = `
<div>
  <a
    href="/"
  >
    <p
      class="font-bold text-[#53B175] text-lg"
    >
      T-Online
    </p>
  </a>
</div>
`;
